sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -16,15 +16,17 @@ export function useFormioDropdownLoader() {
   const loadDropdown = async (
     form: any,
     dropdownName: string,
-    loadMethod: Promise<OptionItemAPIOutDTO[]>,
+    loadMethod: Promise<OptionItemAPIOutDTO[]> | OptionItemAPIOutDTO[],
```

Comment:
I think we don't need this change.

Reply:
typescript error shows up without it.

Reply:
OK. I see the issue. Thanks for checking that.